src/website/app/demos/Popover/bestPractices.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import { createStyledComponent } from '../../../../utils';
+import Button from '../../../../Button';
+import Heading from '../../Heading';
+import Popover from '../../../../Popover';
+
+const styles = {
+  badDemo: {
+    width: '50em'
+  },
+  demoLayout: {
+    display: 'flex',
+    justifyContent: 'space-between'
+  },
+  goodDemo: { width: '15em' },
+  p: ({ theme }) => ({
+    marginBottom: 0,
+    marginTop: theme.spacing_single
+  })
+};
+
+const GoodDemo = createStyledComponent('div', styles.goodDemo);
+const DemoLayout = createStyledComponent('div', styles.demoLayout);
+const P = createStyledComponent('p', styles.p);
+
+class PopOverDo extends React.Component {
```

Comment:
I realize it's a lot, but I think it's potentially confusing to have this practice behave differently than the controlled example (the Popover content doesn't dismiss on document click, for example). Probably best to just use all the hairy code we did there so it matches.

Reply:
addressed

Reply:
It behaves the same as it did when I first reviewed...